src/fklearn/resources/VERSION

```diff
@@ -1 +1 @@
-2.3.1
+2.4.0
```

Comment:
Since removing support to python 3.6 is a breaking change, can you change this version to a major bump?
```suggestion
3.0.0
```